test/plugin/scenarios/resttemplate-6.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+- serviceName: resttemplate-6.x-scenario
+  segmentSize: ge 2
+  segments:
+  - segmentId: not null
+    spans:
+    - operationName: http head /resttemplate/case/healthcheck
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: nq 0
+      endTime: nq 0
+      componentId: nq 0
+      isError: false
+      spanType: Entry
+      peer: '/resttemplate-6.x-scenario/resttemplate/case/healthcheck'
+      tags:
+      - {key: http.url, value: '/resttemplate-6.x-scenario/resttemplate/case/healthcheck'}
+      - {key: method, value: HEAD}
+      - {key: status, value: '200'}
+      skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: http get /resttemplate/syncback
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: nq 0
+      endTime: nq 0
+      componentId: nq 0
+      isError: false
+      spanType: Entry
+      peer: '/resttemplate-6.x-scenario/resttemplate/syncback'
+      tags:
+      - {key: http.url, value: '/resttemplate-6.x-scenario/resttemplate/syncback'}
+      - {key: method, value: GET}
+      - {key: status, value: '200'}
+      skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: http get
+      parentSpanId: 0
+      spanId: 1
+      spanLayer: Http
+      startTime: nq 0
+      endTime: nq 0
+      componentId:
+      isError: false
+      spanType: Exit
+      peer: http://localhost:8080/resttemplate-6.x-scenario/resttemplate/syncback
```

Comment:
Peer should be remote addr only. Using `http.url` is working, but it increases the backend analysis payload a lot.